fathom/src/core.rs

```diff
@@ -432,9 +437,9 @@ pub enum UIntStyle {
     Ascii,
 }
 
-/// Constants
+/// Constant literals
 #[derive(Debug, Copy, Clone, PartialOrd)]
-pub enum Const {
+pub enum ConstLit {
```

Comment:
Why not just `Const`?

Reply:
To distinguish it from the `ConstType`s… which don't currently exist 😳… they are currently implemented on the the `Prim`s which makes everything a bit disjointed. I can revert to the old naming though.